### PR TITLE
docs: update strategy constraints for 4.3

### PR DESCRIPTION
**Pull request by thomasheartman**
This PR updates the strategy constraints documentation for 4.3. It updates the relevant screenshots and rewrites some of the content to reflect that constraining on environment is no longer recommended.

### CODE REVIEW

**Comment by thomasheartman** [comment]
[Document new toggle view](https://trello.com/c/LUTgWZpn/605-document-new-toggle-view)

File: website/docs/advanced/strategy-constraints.md [hunk 1]
```diff
@@ -4,51 +4,70 @@ title: Strategy Constraints
 ---
 
 <div class="alert alert--info" role="alert">
-  Strategy constraints part of Unleash Pro and Enterprise.
+  Strategy constraints are part of Unleash Pro and Enterprise.
 </div>
-<br />
+<BR />
```
Comment (chriswk) [inline website/docs/advanced/strategy-constraints.md:9]: Why uppercase - Something to do with how docusaurus parses our markdown?

File: website/docs/advanced/strategy-constraints.md [hunk 2]
```diff
@@ -4,51 +4,70 @@ title: Strategy Constraints
...
+define all possible values for that context field. These values appear in the Unleash Admin UI to guide users when working with context fields to make sure they only use legal values.
 
-#### What is “legal values”? {#what-is-legal-values}
+![A modal to define constraints. The \"region\" context field is selected and a dropdown is showing the legal values defined for that field: Africa, Asia, Europe, North-America.](/img/constraints_legal_values.png)
 
-Legal values defines all possible values for the context field. this will be used in Unleash Admin UI to guide users when working with context fields to make sure they only use legal values.
+### [Deprecated]: Constrain on a specific environment {#constrain-on-a-specific-environment}
 
-![New context fields](/img/constraints_legal_values.png)
+Before Unleash 4.3, using strategy constraints was the recommended way to have different toggle configurations per environment. Now that Unleash has environment support built in, we no longer recommend you use strategy constraints for this. Instead, see the [environments documentation](../user_guide/environments).
```
Comment (chriswk) [inline website/docs/advanced/strategy-constraints.md:73]: :+1:

**Review by chriswk — APPROVED** [review]
LGTM

File: website/docs/advanced/strategy-constraints.md (hunk 1 above)
Comment (thomasheartman) [reply website/docs/advanced/strategy-constraints.md:9]: Nope, not at all. I was surprised to see this too, must've been a misplaced editor command. I'll fix it though 🙋🏼

Comment (thomasheartman) [reply website/docs/advanced/strategy-constraints.md]: Fixed in aa80cba2221588d75b820db17bf2884e2d752eab

**Pull request merged**